CHANGELOG.md

```diff
@@ -3,6 +3,7 @@
 - Fixed: bug causing rules in extended configs to be merged with, rather than replaced by, the extending config.
 - Fixed: bug where fractional keyframes yielded false positives in `selector-class-pattern`
 - Fixed: `selector-no-type` with secondary option `ignore: ["descendant"]` will now resolve nested selectors.
+- Fixed: `selector-max-specificity` now ignores selectors containing pseudo-classes with functional notation like `:unknown()`, and warns if the underlying `specificity` module cannot parse the selector. 
```

Comment:
I think what, judging from the tests, we have at the moment is:

- `matches` *is explicitly ignored* i.e. won't produce any warnings.
- Anything else that the `specificity` module cannot parse adds a PostCSS warning, this happens to include unknown pseudo-classes with functional notation like `:unknown()`.

It feels like we're going round and around on this one, but I think we're almost there :)